packages/sanity/src/core/studio/components/navbar/search/components/filters/filter/inputs/date/datePicker/calendar/Calendar.tsx

```diff
@@ -0,0 +1,324 @@
+import {Box, Button, Flex, Select, Text, useForwardedRef} from '@sanity/ui'
```

Comment:
Lots of commented out code in this file, intentional?

Reply:
Not intentional!